gatsby-node.js

```diff
@@ -59,6 +59,7 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
 
   // Create Pages
   const { createPage } = actions;
+  const integrationPostTemplate = path.resolve("src/templates/integrations.js");
```

Comment:
Always add new variables to the end of older definitions. And, keep the definition style similar to others
i.e, 
```
       const integrationTemplate = path.resolve(
           "src/templates/integrations.js");
       );
```
P.s: This is not a "post" so just "integrationsTemplate" is fine